src/BaselineUtils.cc

```diff
@@ -883,26 +976,24 @@ symmetrizeFootprint(
     det::Footprint const& foot,
     int cx, int cy) {
 
-    typedef typename det::Footprint::SpanList SpanList;
-
-    PTR(det::Footprint) sfoot(new det::Footprint(foot.getPeaks().getSchema()));
-    const SpanList spans = foot.getSpans();
-    assert(foot.isNormalized());
+    auto sfoot = std::make_shared<det::Footprint>();
+    sfoot->setPeakSchema(foot.getPeaks().getSchema());
```

Comment:
I will note that in general this probably will not be an issue in the future as you would do things differently. I was trying to shoehorn the new code in as exactly as I could into the old pattern